src/reducers/currentServerUrl.js

```diff
@@ -41,6 +42,10 @@ export const currentServerUrl = (state = null, { type, payload }) => {
 		case SIDE_BAR_ADD_NEW_SERVER_CLICKED:
 			return null;
 
+		case SIDE_BAR_DOWNLOADS_BUTTON_CLICKED:
+
+			return 'Downloads';
```

Comment:
I think it's better to reserve another key in the Redux store for the selected panel, as `currentServerUrl` can be used in any point of the codebase as a real URL or `null`; something like `view`, which values can be `server-webview` and `downloads`. Just add a new reducer.